python/graphstorm/config/argument.py

```diff
@@ -747,6 +748,142 @@ def eval_fanout(self):
+                "for example, a valid input can be 'query:query@query,asin:asin@item_name/brand'"
+
+        return dt_feats_dict
+
+    @property
+    def distill_feats(self):
+        """ distillation features
+        """
+        if hasattr(self, "_distill_feats"):
+            dt_feats = self._distill_feats.split(",")
+            return self._check_distill_feats(dt_feats)
+        return None
+
+    @property
+    def textual_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_path"):
+            return self._textual_path
+        return None
+
+    @property
+    def concat_field_name(self):
+        """ filed name for distillation
+        """
+        if hasattr(self, "_concat_field_name"):
+            return self._concat_field_name
+        return None
+
+    @property
+    def tsf_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_tsf_name"):
+            return self._tsf_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
+        """
+        if hasattr(self, "_pre_trained_name"):
+            return self._pre_trained_name
+        return None
+
+    @property
+    def distill_batch_size(self):
+        """ Batch size for distillation.
+        """
+        if hasattr(self, "_distill_batch_size"):
+            return self._distill_batch_size
+        return None
+
+    @property
+    def need_infer(self):
+        """ Whether to run GNN inference for distillation.
+        """
+        if hasattr(self, "_need_infer"):
+            return self._need_infer
+        return None
+
+    @property
+    def save_student_path(self):
+        """ Saved model path for distillation.
+        """
+        if hasattr(self, "_save_student_path"):
+            return self._save_student_path
+        return None
+
+    @property
+    def distill_lr(self):
+        """ Learning rate for distillation.
+        """
+        if hasattr(self, "_distill_lr"):
+            return self._distill_lr
+        return None
+
+    @property
+    def save_student_interval(self):
+        """ Model checkpoint interval for distillation.
+        """
+        if hasattr(self, "_save_student_interval"):
+            return self._save_student_interval
+        return None
+
+    @property
+    def eval_student_interval(self):
+        """ Validation interval for distillation.
+        """
+        if hasattr(self, "_eval_student_interval"):
+            return self._eval_student_interval
+        return None
+
+    @property
+    def chunk_size(self):
+        """ Size of each file for distillation data.
+        """
+        if hasattr(self, "_chunk_size"):
+            return self._chunk_size
+        return None
+
+    def _check_max_global_step(self, max_global_step):
```

Comment:
What does this mean?

Reply:
user can specify the number of global steps for distillation training. Otherwise the training will go infinitely.